lib/CMakeLists.txt

```diff
@@ -127,6 +127,17 @@ endforeach(item ${QUDA_OBJS})
 
 list(REMOVE_ITEM QUDA_OBJS ${QUDA_CU_OBJS})
 
+set(QUDA_MDW_FUSED_LS_LIST "4,8,12,16,20,24" CACHE STRING "The list of Ls the get compiled")
```

Comment:
I think we should mark this as an advanced param

Reply:
Should we only do that processing if DWF is enabled ?

Reply:
https://github.com/lattice/quda/pull/1224/commits/4856c65ef204caf21bf61caad1350ee036945a02

Reply:
@mathiaswagner https://github.com/lattice/quda/pull/1224/commits/c0f74b86e8bc0a7d350aa5f51fb64775d3d09986